public/apps/audit/components/main/EditorBox.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { EuiCodeEditor, EuiText, EuiTextColor } from '@elastic/eui';
+import { get } from 'lodash';
+
+import 'brace/theme/textmate';
+import 'brace/mode/json';
+
+function EditorBox({ setting, config, handleChange, handleInvalid }) {
+  const [value, updateValue] = useState(JSON.stringify(get(config, setting.path), null, 2));
+  const [invalid, setInvalid] = useState(false);
+
+  const onChange = value => {
```

Comment:
Better to have a different variable name since "value" has been used in the larger scope at line#10

Reply:
Changed the variable name to `text` from EditorBox to avoid conflict with `value`